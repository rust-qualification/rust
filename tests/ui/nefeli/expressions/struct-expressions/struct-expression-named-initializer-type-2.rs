// Struct Expressions: f3930897-1745-4c56-9481-be35348298d5

#[allow(dead_code)]
struct Employee {
    name: String,
    age: u16,
    salary: u32,
}

fn main() {
    let maria = Employee {
        name: "maria".to_string(),
        age: "maria".to_string(),
        salary: 50_000,
    };
    //~^ ERROR mismatched types
}