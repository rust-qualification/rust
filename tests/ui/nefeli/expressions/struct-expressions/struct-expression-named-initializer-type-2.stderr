error[E0308]: mismatched types
  --> $DIR/struct-expression-named-initializer-type-2.rs:13:14
   |
LL |         age: "maria".to_string(),
   |              ^^^^^^^^^^^^^^^^^^^ expected `u16`, found `String`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0308`.
